map/styled_point.cpp

```diff
@@ -0,0 +1,54 @@
+#include "map/styled_point.hpp"
+
+#include "base/stl_add.hpp"
+#include "base/string_utils.hpp"
+
+#include "std/fstream.hpp"
+#include "std/algorithm.hpp"
+#include "std/auto_ptr.hpp"
+
+
+graphics::DisplayList * StyledPoint::GetDisplayList(UserMarkDLCache * cache) const
+{
+  return cache->FindUserMark(UserMarkDLCache::Key(GetStyle(), graphics::EPosAbove, GetContainer()->GetDepth()));
```

Comment:
Original ApiPoint mark should be drawed with graphics::ECenter. With EPosAbove pixelPoint{Sprite.Width / 2, Sprite.Height} will be placed on Org point and original Api style will be a little upper that it's need.